analyses/focal-cn-file-preparation/run-prepare-cn.sh

```diff
@@ -44,58 +47,61 @@ Rscript --vanilla 04-prepare-cn-file.R \
   --filename_lead "consensus_seg_annotated_cn" \
   --seg
 
-# libraryStrategies=("polya" "stranded")
-# chromosomesType=("autosomes" "x_and_y")
-# for strategy in ${libraryStrategies[@]}; do
-# 
-#   for chromosome_type in ${chromosomesType[@]}; do
-# 
-#     Rscript --vanilla rna-expression-validation.R \
-#       --annotated_cnv_file results/consensus_seg_annotated_cn_${chromosome_type}.tsv.gz \
-#       --expression_file ${data_dir}/pbta-gene-expression-rsem-fpkm-collapsed.${strategy}.rds \
-#       --independent_specimens_file $independent_specimens_file \
-#       --metadata $histologies_file \
-#       --goi_list $goi_file \
-#       --filename_lead "consensus_seg_annotated_cn"_${chromosome_type}_${strategy}
-#   done
-# done
-# 
-# # if we want to process the CNV data from the original callers
-# # (e.g., CNVkit, ControlFreeC)
-# if [ "$RUN_ORIGINAL" -gt "0" ]; then
-# 
-#   # Prep the CNVkit data
-#   Rscript --vanilla -e "rmarkdown::render('01-add-ploidy-cnvkit.Rmd', clean = TRUE)"
-# 
-#   # Run annotation step for CNVkit
-#   Rscript --vanilla 04-prepare-cn-file.R \
-#     --cnv_file ${scratch_dir}/cnvkit_with_status.tsv \
-#     --gtf_file $gtf_file \
-#     --metadata $histologies_file \
-#     --filename_lead "cnvkit_annotated_cn" \
-#     --seg
-# 
-#   # Run annotation step for ControlFreeC
-#   Rscript --vanilla 04-prepare-cn-file.R \
-#     --cnv_file ${data_dir}/pbta-cnv-controlfreec.tsv.gz \
-#     --gtf_file $gtf_file \
-#     --metadata $histologies_file \
-#     --filename_lead "controlfreec_annotated_cn" \
-#     --controlfreec
-# 
-#   filenameLead=("cnvkit_annotated_cn" "controlfreec_annotated_cn")
-#   for filename in ${filenameLead[@]}; do
-#     for strategy in ${libraryStrategies[@]}; do
-#       for chromosome_type in ${chromosomesType[@]}; do
-#         Rscript --vanilla rna-expression-validation.R \
-#           --annotated_cnv_file results/${filename}_${chromosome_type}.tsv.gz \
-#           --expression_file ${data_dir}/pbta-gene-expression-rsem-fpkm-collapsed.${strategy}.rds \
-#           --independent_specimens_file $independent_specimens_file \
-#           --metadata $histologies_file \
-#           --goi_list $goi_file \
-#           --filename_lead ${filename}_${chromosome_type}_${strategy}
-#       done
-#     done
-#   done
-# 
-# fi
+# Define most focal units of recurrent CNVs
```

Comment:
Why was this commented out previously?

Reply:
For development reasons, it is still relevant for the production of the files that were added to this module previous to this PR.